examples/contracts-and-usage-policy/templates/UsageLoggingTemplates/USAGE_LOGGING_AGREEMENT_TEMPLATE.jsonld

```diff
@@ -17,7 +17,14 @@
             "@type": "ids:Duty",
             "ids:assigner": (idsc:PARTICIPANT_TEMPLATE),          // same value as ids:provider
             "ids:assignee": (idsc:PARTICIPANT_TEMPLATE),          // same value as ids:consumer
-            "ids:action": { "@id": "idsc:LOG" }
+            "ids:action": { 
+				"@id": "idsc:LOG",,
+				"refinement": [{    
+					"leftOperand": "idsc:LoggingEndpoint",  
```

Comment:
idsc:LoggingEndpoint is not in the IM, so only idsc:ENDPOINT?

Reply:
I suggest idsc:SYSTEM_DEVICE (similar to ODRL) and we use it when the logging happens locally to address the log system.